util/text.py

```diff
@@ -29,8 +29,17 @@ class Text:
         "en_US",  # English (US)
     ]
 
+    MALE = "Male"
+    FEMALE = "Female"
```

Comment:
Is it worth having a neuter for German and other languages with 3 grammatical genders?

Reply:
We can add in whatever is necessary for any language when the translators tell us what the languages rules are. I just wanted to lay the groundwork for the parsing in now and added in things that I knew some other languages would need.